src/libs/weather.ts

```diff
@@ -48,88 +21,107 @@ export class WeatherAPI {
 
 }
 
-/**
- * API Parameters and Responses
- */
-
-export interface GetWeatherResponse {
+export interface WeatherResponse {
 	weather: Weather;
 }
 
-/**
- * Helpers
- */
-
-export interface Weather {
-	latitude: number;
-	longitude: number;
-	timezone: string;
-	currently: HourlyWeatherSnapshot;
-	minutely: WeatherDataGroup<MinutelyWeatherSnapshot>;
-	hourly: WeatherDataGroup<HourlyWeatherSnapshot>;
-	daily: WeatherDataGroup<DailyWeatherSnapshot>;
-	flags: {
-		sources: string[];
-		'nearest-station': number;
-		units: string;
-	};
-	offset: number;
+export interface Weather { // all in fahrenheit, miles/hour, imperial
+	temperature: number;
+	temperatureHigh: number;
+	temperatureLow: number;
+	humidity: number;
+	percipitationChance: number;
+	windSpeed: number;
+	windDir: number;
+	weatherIcon: string;
 }
 
-export interface SummaryIcon {
-	summary: string;
-	icon: string;
+// note there is a "minutely" array, but our Open Weather request excludes it
+export interface OpenWeather {
```

Comment:
Since API consumers won't need this, can you move the OpenWeather types to the backend?